packages/builders/src/util/equatable.ts

```diff
@@ -0,0 +1,3 @@
+export interface Equatable<T> {
```

Comment:
Whats this?

Reply:
It's similar to JSONEncodable, it just signals that a builder supports `.equals`